services/admin/src/admin/authentication.py

```diff
@@ -50,7 +33,8 @@ def auth_check(
     if organization is None or external_auth_url is None:
         return True
     try:
-        response = requests.get(external_auth_url, auth=RequestAuth(request), timeout=hf_timeout_seconds)
+        async with httpx.AsyncClient() as client:
+            response = await client.get(external_auth_url, auth=RequestAuth(request), timeout=hf_timeout_seconds)
```

Comment:
this means that we now pass the "cookie", while we didn't before. I think it's OK though

Reply:
Yes, I thought it won't break anything...